homework/calculate/calculate.hpp

```diff
@@ -2,6 +2,31 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    int num_result {0};
+
+    if(command == "add")
+    {
+        num_result = first + second;
+    }
+    else if(command == "subtract")
+    {
+        num_result = first - second;
+    }
+    else if(command == "multiply")
+    {
+        num_result = first * second;
+    }
+    else if(command == "divide")
+    {
+        if(second == 0)
+        {
+            return "Division by 0";
+        }
+        num_result = first / second;
+    }
+    else
+    {
+        return "Invalid data";
+    }
+    return std::to_string(num_result);
```

Comment:
You can also return immediately values :) Both option is good. I prefer immediately return, but this is my preference :)